packages/alchemy/src/client/internal/smartAccountClientFromRpc.ts

```diff
@@ -45,7 +49,15 @@ export function createAlchemySmartAccountClientFromRpcClient({
   gasEstimator,
   customMiddleware,
   client,
-}: CreateAlchemySmartAccountClientFromRpcClient): AlchemySmartAccountClient {
```

Comment:
this override as well should use the default generic values and not also be generic on these types